db/migrations/0014_m2_5_upgrades.sql

```diff
@@ -5,20 +5,17 @@ DROP TABLE indexer_roots;
 DROP TABLE proof_set_stats;
 DROP TABLE retrieval_logs;
 
-CREATE TABLE providers (
+CREATE TABLE service_providers (
   id TEXT NOT NULL,
-  beneficiary_address TEXT NOT NULL,
   service_url TEXT,
   PRIMARY KEY (id)
 );
-CREATE INDEX providers_beneficiary_address ON providers(beneficiary_address);
 
 CREATE TABLE data_sets (
   id TEXT NOT NULL,
-  storage_provider_address TEXT,
-  payer_address TEXT,
-  payee_address TEXT,
```

Comment:
Without `payee` and `storage_provider_address` we're unable to scan them again the sanctions API.